distribution/server/src/assemble/bin.xml

```diff
@@ -131,6 +131,8 @@
         <exclude>io.netty:netty-transport-native-epoll</exclude>
         <exclude>io.netty:netty-transport-native-unix-common</exclude>
 
+        <exclude>org.apache.pulsar:pulsar-functions-runtime-all</exclude>
```

Comment:
Couldn't the runtime-all module just be deleted at this point?

Reply:
The point of having it in the pom is to make sure that the pulsar-functions-runtime-all module is built before this module, which we still need it to